src/Microsoft.AspNet.Mvc.Core/OutputFormatter.cs

```diff
@@ -23,6 +23,19 @@ public abstract class OutputFormatter : IOutputFormatter, IApiResponseFormatMeta
         private readonly List<MediaTypeHeaderValue> _supportedMediaTypes;
 
         /// <summary>
+        /// Returns UTF8 Encoding without BOM and throws on invalid bytes.
+        /// </summary>
+        public static readonly Encoding UTF8EncodingWithoutBOM
```

Comment:
cc @kichalla 

Do we care about being particular about the encoding for output formatters? The BOM specification shouldn't apply since you made your change and we wouldn't be producing invalid bytes to throw, would we?